pkg/ccl/backupccl/utils_test.go

```diff
@@ -197,6 +201,9 @@ func backupRestoreTestSetupEmptyWithParams(
 	ctx := logtags.AddTag(context.Background(), "backup-restore-test-setup-empty", nil)
 
 	params.ServerArgs.ExternalIODir = dir
+	if params.ServerArgs.Knobs.JobsTestingKnobs == nil {
```

Comment:
just trying to wrap my head around this change: if the user doesn't pass any job testing testing knobs, we automatically use testing knobs with short intervals? Maybe amend the commit message to reflect this. Something like: "by default, all tests will have the the jobs.NewTestingKnobsWithShortIntervals() set"